docs/supported-platforms.md

```diff
@@ -16,6 +16,7 @@ Ignition is currently only supported for the following platforms:
 * [Exoscale] (`exoscale`) - Ignition will read its configuration from the instance userdata. Cloud SSH keys are handled separately.
 * [Google Cloud] (`gcp`) - Ignition will read its configuration from the instance metadata entry named "user-data". Cloud SSH keys are handled separately.
 * [IBM Cloud] (`ibmcloud`) - Ignition will read its configuration from the instance userdata. Cloud SSH keys are handled separately.
+* [KubeVirt] (`kubevirt`) - Ignition will read its configuration from the instance userdata via config drive. SSH keys can be injected with the userdata.
```

Comment:
"handled separately" is code for "by Afterburn".  Since the Afterburn PR appears to support those, let's use the typical language here.